src/components/UI/Overlay/OverlayContent/AddressBook.tsx

```diff
@@ -67,7 +67,7 @@ export const AddressBook = styled(({ onAddressSelected, ...props }: AddressBookP
   const [addressBookThirdPartyResults, setAddressBookThirdPartyResults] = useState<AddressBookThirdPartyResultsProps[]>(
     []
   );
-  const { name, endpoint, apiHeader, apiKey } = thirdPartyAPIEndpoints[remoteEndpointIndex];
+  const { name, endpoint, apiHeader, apiKey } = thirdPartyAPIEndpoints[remoteEndpointIndex] ?? [];
```

Comment:
is the ?? [] even the right type? shouldnt it be `?? {}`?

Reply:
thirdparty 1 is [], local one is {}
 
![Screenshot 2020-10-09 at 5 11 51 PM](https://user-images.githubusercontent.com/4774314/95565296-8cc13380-0a52-11eb-96e1-f4f237b97d19.png)

Reply:
oops yes right